docs/stylesheets/extra.css

```diff
@@ -108,4 +108,10 @@
 [data-md-color-scheme="slate"] {
     /* Override the font color in dark mode - text is too light */
     --md-default-fg-color: rgb(240, 246, 252)
+}
+
+.md-typeset .code > .admonition-title::before,
+.md-typeset .code > summary::before {
+    -webkit-mask-image: var(--md-admonition-icon--code);
```

Comment:
What's this needed for? 

And do we need the WebKit prefix or is prefix less below okay?